profiles/templates/components/work-experience-details.html

```diff
@@ -0,0 +1,29 @@
+<div class="work-exp grid grid-cols-1 sm:grid-cols-2 2xl:w-[50%]">
+  <p class="block text-gray-700 text-sm font-bold mb-2">Company Name</p>
+  <p class="work-exp-organization block text-gray-700 text-sm mb-2">{{ work_exp.organization }}</p>
+  <p class="block text-gray-700 text-sm font-bold mb-2">Location</p>
+  <p class="work-exp-location block text-gray-700 text-sm mb-2">{{ work_exp.location }}</p>
+  <p class="block text-gray-700 text-sm font-bold mb-2">Job Title</p>
+  <p class="work-exp-position block text-gray-700 text-sm mb-2">{{ work_exp.position }}</p>
+  <p class="block text-gray-700 text-sm font-bold mb-2">Dates Worked</p>
+  <p class="work-exp-date block text-gray-700 text-sm mb-2">
+    {{ work_exp.start_date }} - {{ work_exp.end_date }}
+  </p>
+  <p class="block text-gray-700 text-sm font-bold mb-2">Duties/Responsibilities</p>
+  <ul class="work-rsp-list pl-6 mb-3">
+    {% with "work-exp-rsp" as list_class %}
+    {% for item_name in work_exp.bullet_points.all %}
+    {% include 'components/list-item.html' %}
+    {% endfor %}
+    {% endwith %}
+  </ul>
+  <p class="block text-gray-700 text-sm font-bold mb-2">Applied Skills</p>
+  <ul class="work-skill-list pl-6">
+    {% with "work-exp-skill" as list_class %}
+    {% for item_name in work_exp.applied_skills.all %}
+    {% include 'components/list-item.html' %}
```

Comment:
Suggestion (Non-blocking): This looks good, but I think it could be nice to add these inside either `<details><summary>` tag or else use the [Flowbite Accordion](https://flowbite.com/docs/components/accordion/) like you mentioned before. It will be responsive out of the box and will save space on the page. What do you think?

Reply:
Yeah, I could look into doing something like that

Reply:
Okay lets move this on and add it as a new enhancement ticket